docs/plugins/outputs/riemann.asciidoc

```diff
@@ -158,14 +151,12 @@ Any other field set here will be passed to Riemann as an event attribute.
 
 Example:
 [source,ruby]
------
     riemann {
         riemann_event => {
             "metric"  => "%{metric}"
             "service" => "%{service}"
         }
     }
------
 
 `metric` and `ttl` values will be coerced to a floating point value.
 Values which cannot be coerced will zero (0.0).
```

Comment:
REGRESSION: Additional formatting required for proper rendering with --asciidoctor